src/middlewared/middlewared/plugins/alert.py

```diff
@@ -439,6 +439,7 @@ async def _send_alert_changed_event(self, alert):
 
     def _send_alert_deleted_event(self, alert):
         self.middleware.send_event("alert.list", "CHANGED", id=alert.uuid, cleared=True)
+        self.middleware.send_event("alert.list", "REMOVED", id=alert.uuid, cleared=True)
```

Comment:
Do we need `cleared=True` here?

Reply:
I think no, `REMOVED` should be enough to clarify that.